ReSwift-ThunkTests/Tests.swift

```diff
@@ -58,4 +59,36 @@ class Tests: XCTestCase {
         store.dispatch(thunk)
         XCTAssertTrue(thunkBodyCalled)
     }
+    
+    func testExpectThunk() {
+        let thunk = Thunk<FakeState> { dispatch, getState in
+            dispatch(FakeAction())
+            XCTAssertNotNil(getState())
+            DispatchQueue.global().asyncAfter(deadline: DispatchTime.now() + 0.5) {
+                dispatch(AnotherFakeAction())
+                XCTAssertNotNil(getState())
+                XCTAssertNil(getState())
+            }
+            dispatch(FakeAction())
+        }
+        
+        let expect = ExpectThunk<FakeState>(thunk)
```

Comment:
Here's the result of the proposal! Basically a chain-able expectation much like redux saga test plan